docs/reference/inference/service-cohere.asciidoc

```diff
@@ -73,7 +73,11 @@ More information about Cohere's rate limits can be found in https://docs.cohere.
 .`service_settings` for the `completion` task type
 [%collapsible%closed]
 =====
-
+`model_id`::
+(Optional, string)
+The name of the model to use for the {infer} task.
+To review the available `rerank` models, refer to the
```

Comment:
Let's update `rerank` here to `completion` and update the doc link below.

Reply:
Nice catch, thanks!
However, I haven't found a page on Cohere's side to link to. Am I missing something?

Reply:
I think these are the chat completion models: https://docs.cohere.com/docs/models#command

Reply:
@jonathan-buttner Thank you, I updated the PR via [d8769ec](https://github.com/elastic/elasticsearch/pull/109812/commits/d8769ec05c06443dae5b1505ef527846ea1a3f06).